src/Symfony/Component/HttpFoundation/Session/Storage/Handler/StrictSessionHandler.php

```diff
@@ -28,6 +34,15 @@ public function __construct(\SessionHandlerInterface $handler)
         }
 
         $this->handler = $handler;
+        $this->wrapper = $handler instanceof \SessionHandler;
+    }
+
+    /**
+     * Returns true if this handler wraps an internal PHP session save handler using \SessionHandler.
+     */
+    public function isWrapper(): bool
```

Comment:
We usually try to avoid adding new public methods in bugfix releases, but I see why this might be necessary here. But can we please make this method `final` and flag it as `@internal`?